### ISSUE TITLE
Reformat code

### ISSUE THREAD

**Issue by passsy** [guylabs/intellij-idea-keymap#6]
I personally added the shortcut to reformat code such as `JSON`. 

``` diff
  ###########
  # Editing #
  ###########

  # ...

+
+  # Reformat Code using the beautify plugin
+  'alt-cmd-l': 'atom-beautify:beautify-editor'

  # ...
```

Since atom doesn't support reformatting out of the box a dependency to [atom-beautify](https://atom.io/packages/atom-beautify) is required. I'm not sure if this is the scope of this project

**Comment by guylabs** [comment]
Hi @passsy 

well this would be a cool addition. I will check that and try to integrate it.

Thanks and regards,

Guy

**Comment by guylabs** [comment]
Hi @passsy,

thanks to @5pid this is also fixed in the #7 pull request which was merged minutes ago. I will update the release documents and then release a new version with the changes.

Thanks and regards,

Guy
